dropwizard-bom/pom.xml

```diff
@@ -24,8 +24,9 @@
         <dropwizard.version>${project.version}</dropwizard.version>
         <guava.version>19.0</guava.version>
         <jersey.version>2.22.2</jersey.version>
-        <jackson.api.version>2.6.0</jackson.api.version>
-        <jackson.version>2.6.5</jackson.version>
+        <jackson.api.version>2.7.3</jackson.api.version>
+        <jackson.version>2.7.3</jackson.version>
+        <jackson.jaxrs.version>2.7.2</jackson.jaxrs.version>
```

Comment:
Fair enough. @cowtowncoder, could you please publish a release `jackson-jaxrs-json-provider` with 2.7.3 version? It will make supporting Dropwizard a little bit easier.